README.md

```diff
@@ -20,7 +20,7 @@ resources:
   index.html:
     type: aws:s3:BucketObject
     properties:
-      bucket: ${my-bucket}
+      bucket: ${my-bucket.id}
```

Comment:
Ensuring this behavior works I think is important to the ergonomics of Pulumi YAML.